app/models/app.rb

```diff
@@ -2,19 +2,23 @@ class App < ActiveRecord::Base
   belongs_to :org
   has_many :engagements
   has_many :iterations, :through => :engagements
-  
-  validates_presence_of :name, :description, :org_id, :status, :repository_url
-  
+
+  validates_presence_of :name, :description, :org_id, :status
+  validates_presence_of :repository_url, unless: :pending?
 
   default_scope { order(:name => :asc) }
-  enum :status => { :dead => 0, :development => 1, :in_use => 2, :in_use_and_wants_improvement => 3, :inactive_but_wants_improvement => 4 }
+  enum :status => { :pending => 0, :dead => 1, :development => 2, :in_use => 3, :in_use_and_wants_improvement => 4, :inactive_but_wants_improvement => 5}
 
-  scope :featured, -> { where.not("status = 0") }
+  scope :featured, -> { where.not("status = 0 or status = 1") }
```

Comment:
i realize the initial commit of this line was me, but if there was a way to use the enum symbols here rather than integers that would be much better

Reply:
fixed